src/Text/KirbyTags.php

```diff
@@ -20,23 +21,25 @@
  */
 class KirbyTags
 {
-    protected static $tagClass = 'Kirby\Text\KirbyTag';
-
-    public static function parse(string $text = null, array $data = [], array $options = []): string
+    public static function parse(string $text = null, array $data = [], array $options = [], ?App $app = null): string
     {
+        if ($app !== null) {
```

Comment:
I think we should handle the hooks in the App class in the kirbytags method instead. We already do that in the kirbytext method. I'm not sure why we went this way in the extended class in the first place.

Reply:
moved them